elasticsearch/build.gradle.kts

```diff
@@ -20,6 +20,12 @@ plugins {
     id("com.github.vlsi.gradle-extensions")
 }
 
+repositories {
+    maven {
+        setUrl("https://maven.codelibs.org/")
```

Comment:
Why do we need to add a repository? It seems like we already source some stuff from org.codelibs, is this new things now available in the existing repositories?

Reply:
After ES 7.10.2, "org.codelibs.elasticsearch.module:lang-painless" is no longer maintained in central maven repo, which migrated to https://maven.codelibs.org/, so does "org.codelibs.elasticsearch.module:scripting-painless-spi".
That's why I added a new repo.(see https://github.com/codelibs/elasticsearch-module#version)